src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -595,6 +695,7 @@ protected Locator locator()
 
         public Select getFromSchemaInput()
         {
+            expand();
             Select select = SelectWrapper.Select(Locator.name("domainpropertiesrow-lookupSchema")).find(this);
             getWrapper().waitFor(()-> select.getOptions().size() > 0 && !select.getOptions().get(0).getText().equals("Loading..."),
```

Comment:
This `expand` call looks redundant; everywhere that calls this method does their own expand first. Same for `getFromTargetTableInput` and `getLookupValidatorEnabledCheckbox` below. Its also inconsistent with all of the other elements in the ElementCache